src/main/scala/org/broadinstitute/dsde/snoop/ws/SnoopApiFormats.scala

```diff
@@ -8,27 +10,50 @@ class SnoopApiFormats {
 
 case class WorkflowExecution(
     id: Option[String], 
-    workflowParameters: Map[String, String],
+    workflowParameters: Map[String, WorkflowParameter],
     workflowId: String, 
     callbackUri: String,
     status: Option[String])
 
-case class ZamboniWorkflow(
-    Zamboni: Map[String, String],
-    workflow: Map[String, String])
-
-case class ZamboniSubmission(
-    authToken: String,
-    requestString: String)
-
-case class ZamboniSubmissionResult(
-    workflowId: String,
-    status: String)
-
+/**
+ * Special object to encompass Union-like behavior of a workflow parameter:
+ * a workflow parameter can be either a string or a list of strings but not
+ * both at the same time. Valid json would look like:
+ * "workflowParameters": { "param": "value", "paramList": [ "value1", "value2" ] }
+ * @param value
+ * @param list
+ */
+case class WorkflowParameter(value: Option[String], list: Option[List[String]]) {
```

Comment:
Also if there's no reason for it to be List might as well use Seq. Vectors are generally better than Lists in terms of concrete Seq, but Seq allows to use whatever